sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md

```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 1.5.1 (Unreleased)
+## 1.5.1 (2024-03-07)
```

Comment:
Is this a preview release or stable release?

Reply:
Preview. I'll update version and date.